classes/class-p4-campaigns.php

```diff
@@ -17,6 +17,11 @@ class P4_Campaigns {
 		 * @var string $taxonomy
 		 */
 		private $taxonomy = 'post_tag';
+
```

Comment:
I would prefer all the related custom post type functions (hooks, metabox callbacks etc..) moved to another class.
The class name may be relevant (and confusing), but it is used currently for other purposes, so it is better to separate the cpt to it's own class (maybe P4_CampaignsPostType).

Reply:
Alright will work on this.